styles.css

```diff
@@ -11,7 +11,6 @@ body {
 }
 
 .container {
-  width: 100%;
   margin: auto;
   display: grid;
 }
```

Comment:
- [ ] The width of the page is big, please resize it to fit the page.

![md](https://user-images.githubusercontent.com/61562626/135327101-45913c58-1d88-4431-9e3a-ade4e23ff7b2.png)

Reply:
Adjusted Width in the container